tests/redirects/test_urls.py

```diff
@@ -70,3 +42,46 @@ def test_404_url(base_url):
 @pytest.mark.nondestructive
 def test_x_robots_tag(base_url):
     assert_valid_url(base_url, resp_headers={'x-robots-tag': 'noodp'})
+
+
+@pytest.mark.headless
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('url', [
+    '/firefox/',
+    '/firefox/all/',
+    '/firefox/android/',
+    '/firefox/android/faq/',
+    '/firefox/aurora/all/',
+    '/firefox/beta/all/',
+    '/firefox/brand/',
+    '/firefox/channel/',
+    '/firefox/channel/android/',
+    '/firefox/desktop/',
+    '/firefox/developer/',
+    '/firefox/geolocation/',
+    '/firefox/installer-help/',
+    '/firefox/interest-dashboard/',
+    '/firefox/latest/releasenotes/',
+    '/firefox/mobile/',
+    '/firefox/new/',
+    '/firefox/nightly/firstrun/',
+    '/firefox/organizations/',
+    '/firefox/os/',
+    '/firefox/os/notes/1.1/',
+    '/firefox/partners/',
+    '/firefox/releases/',
+    '/firefox/speed/',
+    '/firefox/sync/',
+    '/firefox/tiles/',
+    '/firefox/unsupported-systems/',
+    '/firefox/unsupported/EOL/',
+    # Legacy URLs (Bug 1110927)
+    'firefox/panorama/',
```

Comment:
The above start with a slash, and the below do not. Only one of these is probably correct.